archive/app/controllers/ArchiveController.scala

```diff
@@ -64,7 +64,7 @@ class ArchiveController(dynamoDB: DynamoDB) extends Controller with Logging with
         Some(path)
       case _ => None
     }
-    s"http://${normalised.getOrElse(path)}"
+    s"https://${normalised.getOrElse(path)}"
```

Comment:
Absolutely. I agree this is confusing and it looks like a side effect of the `normalisation`.
This probably need a bit of tidy up or even a redesign because the redirect table seems to be used for a lot of different concerns :/